.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+sudo: required
+
+python:
+  - "3"
+
```

Comment:
Do we need to create any log directory before_install?

Reply:
no need currently, plz see my the test report.